mmaction/apis/__init__.py

```diff
@@ -1,9 +1,10 @@
 from .inference import inference_recognizer, init_recognizer
```

Comment:
Why do you add the `denseflow` submodule (there is no place to use it)?

Reply:
Apologies, it was from previous debugging/research. Will remove